Source/Module/Main/ContentView.swift

```diff
@@ -14,13 +14,14 @@ struct ContentView: View {
 
     init() {
         _isLoggedIn = State(initialValue: viewModel.getIsLoggedIn())
+        UITableViewCell.appearance().selectionStyle = .none
```

Comment:
I think we can do it by using `List.listStyle()` 🤔

Reply:
@markgravity I set `listStyle` to be `.listStyle(SidebarListStyle())` but the highlight on tap still appear. I also tried this [approach](https://www.hackingwithswift.com/quick-start/swiftui/how-to-disable-the-overlay-color-for-images-inside-button-and-navigationlink) but it doesn't work for some reason. 🥲
```Swift
NavigationLink(
    ...
)
.buttonStyle(PlainButtonStyle())
```